src/lib.rs

```diff
@@ -43,39 +50,40 @@ pub struct Redirect {
     #[serde(default)]
     pub force: bool,
     pub headers: Option<HashMap<String, String>>,
-    #[serde(alias = "params")]
-    #[serde(alias = "parameters")]
+    #[serde(alias = "params", alias = "parameters")]
     pub query: Option<HashMap<String, String>>,
     pub conditions: Option<HashMap<String, HashSet<String>>>,
     pub signed: Option<String>,
-    #[serde(alias = "edge-handler")]
-    pub edge_handler: Option<String>,
 }
 
 /// Header holds information to add response headers for a give url.
-#[derive(Debug, Serialize, Deserialize, PartialEq, Default)]
+#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
 pub struct Header {
     #[serde(rename = "for")]
     pub path: String,
     #[serde(rename = "values")]
     pub headers: HashMap<String, HeaderValues>,
 }
 
-#[derive(Debug, PartialEq, Default)]
+#[derive(Clone, Debug, Default, PartialEq)]
 pub struct HeaderValues {
     pub values: Vec<String>,
 }
 
 /// Template holds information to turn a repository into a Netlify template.
-#[derive(Debug, Serialize, Deserialize, PartialEq, Default)]
+#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
 pub struct Template {
     #[serde(rename = "incoming-hooks")]
     pub hooks: Option<Vec<String>>,
     pub environment: Option<HashMap<String, String>>,
 }
 
-/// Base crate error type
-pub type Error = toml::de::Error;
+/// A mount of an edge handler under a specific path.
+#[derive(Clone, Debug, Default, PartialEq, Serialize, Deserialize)]
+pub struct EdgeHandler {
+    pub handler: String,
+    pub path: String,
+}
```

Comment:
for headers it looks like this right now:
```
[[headers]]
  for = "/*"
  [headers.values]
```

maybe add `for` as an alias for `path`?